ui/panels-plugin/src/index.ts

```diff
@@ -12,41 +12,49 @@
 // limitations under the License.
 
 import { PluginSetupFunction } from '@perses-dev/plugin-system';
-import { EmptyChart, EmptyChartKind } from './plugins/empty-chart/EmptyChart';
-import { GaugeChartPanel, GaugeChartKind } from './plugins/gauge-chart/GaugeChartPanel';
-import { LineChartPanel, LineChartKind } from './plugins/line-chart/LineChartPanel';
-import { StatChartKind, StatChartPanel } from './plugins/stat-chart/StatChartPanel';
+import { createInitialEmptyChartOptions, EmptyChart, EmptyChartOptionsEditor } from './plugins/empty-chart';
+import { createInitialGaugeChartOptions, GaugeChartPanel, GaugeChartOptionsEditor } from './plugins/gauge-chart';
+import { LineChartPanel, LineChartOptionsEditor, createInitialLineChartOptions } from './plugins/line-chart';
+import { createInitialStatChartOptions, StatChartOptionsEditor, StatChartPanel } from './plugins/stat-chart';
 
 export const setup: PluginSetupFunction = (registerPlugin) => {
   registerPlugin({
     pluginType: 'Panel',
-    kind: LineChartKind,
+    kind: 'LineChart',
     plugin: {
       PanelComponent: LineChartPanel,
+      OptionsEditorComponent: LineChartOptionsEditor,
+      createInitialOptions: createInitialLineChartOptions,
```

Comment:
This setup will work basically the same for our "third party" plugins right or are there any differences between them and our 4 core panel plugins?

Reply:
Yup, should be the same for all plugins.